docs/context_standards/README.MD

```diff
@@ -305,6 +305,26 @@ The following is the format for a process.
     "Child": "LIST, of Child process objects"
 }
 ```
+
+## Host
+The following is the format for a host.
+```python
+"Host": {
+    "Domain": "STRING, Domain of the host.",
+    "Hostname": "STRING, Name of the host.",
+    "BIOVersion": "STRING, BIOS version of the host.",
+    "ID": "STRING, The unique ID within the tool retrieving the host.",
+    "DHCPServer": "STRING, DHCP Server.",
+    "IP": "STRING, IP Address of the host.",
```

Comment:
"IP": "STRING, IP address of the host.",

Reply:
@kirbles19 I'll fix that. Except for that, are the descriptions ok?